app/theme/client/imports/components/table.css

```diff
@@ -211,11 +211,14 @@
 	text-align: left;
 
 	color: var(--rc-color-primary-light);
+	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)